scripts/integrations/main.py

```diff
@@ -0,0 +1,89 @@
+import typer
+import configparser
+import json
+import os
+import sys
+import shutil
+from rich import print
+
+app = typer.Typer()
+base_integration_path = "../integrations"
+dirs = ["assets", "info", "samples", "schema", "test"]
+
+
+def createIntegrationDirs(directory, force):
+    if os.path.isdir(directory):
+        if force:
+            force = typer.confirm(f"Are you sure you want to delete {directory}?")
+            if force:
+                shutil.rmtree(directory)
+                os.makedirs(directory)
+                for dir in dirs:
+                    os.makedirs(directory + "/" + dir)
+        else:
+            print(f"[red]Directory {directory} already exists[/red]")
+            sys.exit(1)
+    else:
+        os.makedirs(directory)
+        for dir in dirs:
+            os.makedirs(directory + "/" + dir)
+
+
+def populateDefaultVars(var_name):
+    config = configparser.ConfigParser()
+    config.read('config.ini')
+    return (config.get("DEFAULT", var_name))
+
+
+def generateJson(directory, integration_name, integration_desc, schema, license):
+    data = {
+        "name": integration_name,
+        "version": {
+            "integ": "0.1.0",
+            "schema": schema,
+            "resource": "^1.23.0",
+            "license": license
+        },
+        "description": integration_desc,
+        "identification": "instrumentationScope.attributes.identification",
+        "categories": [],
+        "collection": [{}],
+        "repo": {
+            "github": "https://github.com/opensearch-project/observability/tree/main/integrarions/"+integration_name
+        }}
+
+    with open(directory + '/config.json', 'w', encoding='utf-8') as f:
```

Comment:
Using `os.path.join` is more stable than string concatenation across platforms, see https://docs.python.org/3/library/os.path.html#os.path.join. This also shows up on lines 22, 29, and 75.

Reply:
👍